backends/sycl-ref/ceed-sycl-ref-qfunction-load.sycl.cpp

```diff
@@ -82,7 +82,7 @@ extern "C" int CeedSyclBuildQFunction(CeedQFunction qf) {
   code << "\n";
 
   // Kernel function
-  code << "__kernel void " << kernel_name << "(__global void *ctx, CeedInt Q,\n";
+  code << "__attribute__((intel_reqd_sub_group_size(16))) __kernel void " << kernel_name << "(__global void *ctx, CeedInt Q,\n";
```

Comment:
Let's make this a defined constant at the top of this file for now, with an eye to define it at (library) compile time.